packages/analyzer/src/utils/manifest-helpers.js

```diff
@@ -140,3 +140,25 @@ export function getModuleForClassLike(cem, className) {
   return result;
 }
 
+/**
+ * Given a manifest module, a class name, and a class member name, gets the 
+ * manifest doc for the module's class' member.
+ *
+ * @param  {Partial<import('custom-elements-manifest/schema').Module>} moduleDoc Manifest module
+ * @param  {string} className Class to get member of
+ * @param  {string} memberName Class member to get
+ * @return {import('custom-elements-manifest/schema').ClassMember|void} the requested class member
+ */
+export function getClassMemberDoc(moduleDoc, className, memberName) {
+  /** @type {import('custom-elements-manifest/schema').ClassDeclaration} */
+  const classDoc = (moduleDoc.declarations.find(x => x.name === className));
+
+  if (!classDoc)
+    return console.warn(`Could not find class ${className}`);
+  if (!Array.isArray(classDoc.members))
```

Comment:
we have a `has` helper [here](https://github.com/open-wc/custom-elements-manifest/blob/master/packages/analyzer/src/utils/index.js#L5) could you use that instead?

Reply:
done